bundles/org.openhab.ui/web/src/components/app.vue

```diff
@@ -90,6 +90,9 @@
           <f7-icon slot="media" ios="f7:question_circle_fill" aurora="f7:question_circle_fill" md="material:help" color="gray"></f7-icon>
         </f7-list-item>
       </f7-list>
+      <f7-link class="breakpoint-pin" @click="toggleVisibleBreakpoint">
+        <f7-icon slot="media" size="14" :f7="this.visibleBreakpointDisabled ? 'pin_fill' : 'pin'" color="gray"></f7-icon>
```

Comment:
Actually, for me the filled pin means "pinned". So, I think it should be the other way round: When the breakpoint is disabled the side panel will disappear, meaning the side panel is not-pinned = not filled = 'pin'. What do you think?

```suggestion
        <f7-icon slot="media" size="14" :f7="this.visibleBreakpointDisabled ? 'pin' : 'pin_fill'" color="gray"></f7-icon>
```

Reply:
Yeah the thing is: I'd still like the pinned state to be the default normal state, and unpinned as a non-default alternative. And I'd want the less visible icon to be shown when it's pinned because it'll be always displayed.
And I also don't want a "negative" icon like pin_slash in the default state ;)
So no perfect solutions...
Maybe pinned=pin, unpinned=pin_slash_fill, so that the icon reflects the state, not the action and it's more prominent when it's on the hidden panel?

Reply:
> I'd want the less visible icon to be shown when it's pinned
I'm totally with you.

>  And I also don't want a "negative" icon like pin_slash in the default state ;)
You probably also got that the wrong way when you thought about the slash: pin_slash would "unpinned", so actually not in the default state 😉 
So, seems the accepted solution for us both would be to have "pin" as pinned (=default) and "pin_slash" for side panel hidden... 👍 

```suggestion
        <f7-icon slot="media" size="14" :f7="this.visibleBreakpointDisabled ? 'pin_slash' : 'pin'" color="gray"></f7-icon>
```